wrappers/services.go

```diff
@@ -75,30 +76,109 @@ func generateSnapServiceFile(app *snap.AppInfo, opts *AddSnapServicesOptions) ([
 	return genServiceFile(app, opts)
 }
 
+// max returns the maximum of two integers. Why is this
+// not provided by golang?
```

Comment:
You are not alone, at least: https://github.com/snapcore/snapd/blob/master/strutil/version.go#L27 :-)

Reply:
Haha, it's really wierd basic functions like that are missing!

Reply:
I even have a commit shelved with max and min moved into intutils because I was bored but decided to leave it as it seems overkill indeed